uint8 currentMode
uint16 fault
uint8 counterStatus1

uint8 remote
uint8 realtime
uint8 remoteAccelerator
uint8 realtimeAccelerator
uint16 angle
uint8 countStatus2

uint8 pressure
uint8 brake
uint8 remoteMode
uint8 emergencyStatus
bool remoteStatus
uint8 countStatus3